python/rascaline/calculators.py

```diff
@@ -301,3 +301,48 @@ def __init__(
             parameters["radial_scaling"] = radial_scaling
 
         super().__init__("soap_power_spectrum", parameters)
+
+
+class SoapRadialSpectrum(CalculatorBase):
+    """Radial spectrum of Smooth Overlap of Atomic Positions (SOAP).
+    equal to the Power spectrum of Smooth Overlap of Atomic Positions (SOAP) with l=0.
+
+    The SOAP power spectrum is the main member of the SOAP
+    family of descriptors. The power spectrum is based on the
+    :py:class:`SphericalExpansion` coefficients, which are combined to create a
+    rotationally invariant three-body descriptor.
+
+    See `this review article <https://doi.org/10.1063/1.5090481>`_ for more
+    information on the SOAP representation, and `this paper
+    <https://doi.org/10.1063/5.0044689>`_ for information on how it is
+    implemented in rascaline.
+
+    For a full description of the hyper-parameters, see the corresponding
+    :ref:`documentation <soap-power-spectrum>`.
+    """
```

Comment:
looks like a copy-paste of the power spectrum, which needs to be updated

Reply:
ah sì era quello che avevo scritto all'inizio con radial spectrum equal to power spectrum for l=0